src/codec_svt.c

```diff
@@ -161,6 +161,15 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
             svt_config->profile = HIGH_PROFILE;
         }
 
+        if (encoder->keyframeInterval > 1) {
+            svt_config->intra_period_length = encoder->keyframeInterval - 1;
+        } else if (encoder->keyframeInterval == 1) {
+            // For this case, SV1-AV1 seems to need this flag set in svt_config
```

Comment:
Optional: "Seems to" makes this comment sound speculative. It would be good to have an SVT-AV1 team member review this so we can write this comment with a definitive tone.

Reply:
I have updated the comment to remove the speculation. But like you suggested, we can consult the SVT-AV1 team to make sure this is right.